doc/guides/modules.rst

```diff
@@ -529,10 +587,14 @@ requires the following folder structure:
 .. code-block:: none
 
    <module-root>
+   ├── zephyr/module.yml
    ├── arch
    ├── boards
    ├── dts
-   └── soc
+   ├── soc
+   ├── ...
+   ├── CMakeLists.txt
+   └── Kconfig
```

Comment:
As this code block now contains both files and folders, should we add `/` to the end of folders to indicate what is what ?
```suggestion
   ├── zephyr/module.yml
   ├── arch/
   ├── boards/
   ├── dts/
   ├── soc/
   ├── ...
   ├── CMakeLists.txt
   └── Kconfig
```